app/build.gradle

```diff
@@ -33,15 +33,15 @@ android {
 
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:design:25.1.0'
-    implementation 'com.android.support:gridlayout-v7:25.1.0'
-    implementation 'com.google.android.gms:play-services-drive:10.0.1'
-    implementation 'com.getpebble:pebblekit:3.1.0'
-    implementation 'com.github.JakeWharton:ViewPagerIndicator:2.4.1'
-    implementation 'com.jakewharton.timber:timber:4.3.1'
-    implementation 'com.pavelsikun:material-seekbar-preference:2.3.0+'
-    implementation 'com.github.PhilJay:MPAndroidChart:v3.0.0'
-    implementation "com.github.hotchemi:permissionsdispatcher:2.2.0"
-    annotationProcessor "com.github.hotchemi:permissionsdispatcher-processor:2.2.0"
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
compile instruction deprecated, please update Android studio/graddle

Reply:
That was probably added automatically... or stayed there when I merged with your most current code. Will update.